tsconfig.json

```diff
@@ -19,7 +19,12 @@
     "lib": [
       "es2018",
       "dom"
-    ]
+    ],
+    "paths": {
```

Comment:
Add path alias so we can use `@environment`, `@guard`, `@service` instead of the relative path to those directories when we import them